python/python/lance/torch/kmeans.py

```diff
@@ -263,6 +304,24 @@ def _fit_once(
         )
         return total_dist
 
+    def rebuild_index(self):
+        if self.use_cuvs:
+            rebuild_time_start = time.time()
```

Comment:
would it be easy and clean to have a `cuvs.KMeans`, don't need to share with this Torch code. Mixing cuvs with torch is not maintainable.

Reply:
Done